### PR TITLE
Oses's addition

### PR DESCRIPTION
**Pull request by Oseghale3**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,6 +29,15 @@ <h5 class="card-title">Card title</h5>
 
 </main>
 
+<div class="card" style="width: 18rem;">
```
Comment (charlieboy101) [inline index.html:32]: no comment

**Review by charlieboy101 — COMMENTED** [review]
put more effort

File: index.html (hunk 1 above)
Comment (Oseghale3) [reply index.html:32]: abeg na

Comment (charlieboy101) [reply index.html:32]: nice job boss

**Pull request merged**